lib/onc_certification_g10_test_kit/bulk_export_validation_tester.rb

```diff
@@ -152,13 +155,19 @@ def check_file_request(url) # rubocop:disable Metrics/CyclomaticComplexity
       line_count
     end
 
+    def omit_or_skip(message)
+      omit_if (OMIT_KLASS.include? resource_type), message
+      skip message
```

Comment:
Why is this a skip?  If you they are sending the wrong type of resource in this file, I believe that this is a failure.  We use skip to mean 'haven't sent enough data for us to decide if you should pass', which doesn't apply here.